tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/app/OpenConfigEditor.java

```diff
@@ -10,33 +10,23 @@
  ******************************************************************************/
 package org.jboss.tools.aerogear.ui.bot.test.app;
 
-import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 import org.jboss.reddeer.eclipse.jdt.ui.ProjectExplorer;
 import org.jboss.reddeer.requirements.cleanworkspace.CleanWorkspaceRequirement.CleanWorkspace;
-import org.jboss.reddeer.workbench.impl.editor.DefaultEditor;
 import org.jboss.tools.aerogear.ui.bot.test.AerogearBotTest;
 import org.junit.Test;
 
 @CleanWorkspace
 public class OpenConfigEditor extends AerogearBotTest {
+	
 	@Test
 	public void canOpenConfigXmlEditor() {
-
 		createHTMLHybridMobileApplication(AerogearBotTest.CORDOVA_PROJECT_NAME, AerogearBotTest.CORDOVA_APP_NAME,
 				"org.jboss.example.cordova", "cordova-android@4.1.0");
 		assertTrue(new ProjectExplorer().containsProject(AerogearBotTest.CORDOVA_PROJECT_NAME));
-		openInConfigEditor(CORDOVA_PROJECT_NAME, "config.xml");
-		assertNotNull(new DefaultEditor(CORDOVA_APP_NAME));
-
-	}
+		openInConfigEditor(CORDOVA_PROJECT_NAME, AerogearBotTest.CORDOVA_APP_NAME, "config.xml");
 
-	@Override
-	public void tearDown() {
-		// close config editor before deleting project
-		new DefaultEditor().close();
```

Comment:
no need to different tearDown() method. Editors are closed in super implementation - CleanWorkspace.fulfill() already does this